### ISSUE TITLE
flake8 plugin to enforce f-strings

### ISSUE THREAD

**Issue by peterjc** [ikamensh/flynt#36]
I am aware that ``flynt --fail-on-change`` could be used as part of a git pre-commit hook or continuous integration setup.

My preference for that kind of thing is to use a flake8 plugin. My reading of #24 is that you are not interested in extending flynt itself to offer a flake8 plugin. Is that correct?

If so, are you aware of a third party plugin which you could link to from the README file?

**Comment by peterjc** [comment]
I couldn't find a plugin which did what I wanted, so I've tried writing one:

https://github.com/peterjc/flake8-sfs
https://pypi.org/project/flake8-sfs/

**Comment by ikamensh** [comment]
in #24 idea was to add flake8 and other linters to default dev environment of flynt. We did some of it, but not all. 

On the matter of writing a flake8 plugin, I think it would be best to keep that as a separate project. The only part of Flynt that is relevant there is finding candidate expressions. In principle that could be abstracted into a package of its own, becoming a common dependency of both flynt and the new flake8 plugin.

So far I don't know about any plugins doing this. And good job on writing a plugin of your own! In flynt I have used tokenizer to find beginning and end of all candidate expressions - ast does not show end with python <3.8; I have seen you use ast directly and I guess this is the right approach assuming you don't need to know where expressions end.
